tests/test_kubo_cas.py

```
@@ -0,0 +1,146 @@
+
+    # Your original test code continues here
+    async with KuboCAS(session=global_client_session) as kubo_cas_default:
+        encoded_data = dag_cbor.encode(data)
+        for codec in ["raw", "dag-cbor"]:
+            codec_typed = cast(Literal["raw", "dag-cbor"], codec)
+            try:
+                cid = await kubo_cas_default.save(encoded_data, codec=codec_typed)
+                loaded_encoded_data = await kubo_cas_default.load(cid)
+                result = dag_cbor.decode(loaded_encoded_data)
+                assert data == result
+            except Exception as e:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)